projects/health-warden.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name_cs": "HealthWarden",
+  "name_en": "HealthWarden",
+  "name_de": "HealthWarden",
+  "subtitle_cs": "Proměňte hospitalizované pacienty na ambulantní",
+  "subtitle_en": "Turning inpatients into outpatients",
+  "subtitle_de": "Machen Sie stationäre Patienten zu ambulanten",
+  "website": "https://warden.health",
+  "email": "healthwarden@mild.blue",
+  "desc_cs": "Heart Rate Detector je mobilní aplikace umožňující měřit, vyhodnocovat a sdílet srdeční tep detekovaný kamerou chytrého telefonu. Aplikace umožňuje pacientovi i lékaři sbírat zaznamenaná data a pracovat s nimi s cílem prevence či léčby srdečních onemocnění. Dalšími funkcemi jsou export dat ve formátu CSV, prohlížení historie pacientů nebo zasílání zvacích kódů pro spojení pacientů s lékařem. Aplikace je pro uživatele k dispozici zdarma a umožňuje zkvalitňování a sledování zdravotní péče bez dalších nákladů pro pacienta nebo nemocnici.",
+  "desc_en": "Heart Rate Detector is a mobile app with the possibility to measure, evaluate and share heart pulse detected by a smartphone camera. The application allows both patient and doctor to collect recorded data over time and work with them both on a scale for the doctor taking care of the patient and in an anonymized form on a mass scale. Other functionalities are exporting data in CSV format, viewing patients' history, or sending invitation codes to connect patients with a doctor. The application is available free of charge for users, enabling improvement and monitoring of health care without any additional cost for a patient or a hospital.",
```

Comment:
Other functionalities are data export in CSV format, viewing patient history or sending invitation codes connecting patients and doctors. 

carka pred or je spatne

Reply:
actually grammarly pise ze je tam spravne, ale or predpoklada antitezi takze
Other functionalities are data export in CSV format, viewing patient history, and sending invitation codes connecting patients and doctors.